templates/support_channels.html

```diff
@@ -0,0 +1,94 @@
+{% load i18n %}
+{% load static %}
+
+<section class="frontpage frontpage-section">
```

Comment:
Some things are missing from this page. Should they be added? At the moment the information is not available elsewhere.

* https://github.com/apluslms/mooc-grader/blob/master/courses/README.md
* https://github.com/apluslms/a-plus-rst-tools/blob/master/README.md
* https://github.com/apluslms/a-plus/blob/master/doc/GRADERS.md
* https://github.com/apluslms/a-plus/blob/master/doc/CONTENT.md

Reply:
I think these links should be added in the A+ manual and not here. These links are too specific. If you agree, I can create a new issue in the plus manual for adding this information.

Reply:
OK, aplus-manual. I have been thinking that the manual course front page should list these other important READMEs. The A+ course settings have the introduction field that adds HTML text to the course front page.